cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunNativeImageTestTask.java

```diff
@@ -315,6 +315,9 @@ private int runTestSuiteWithNativeImage(Package currentPackage, JBallerinaBacken
         cmdArgs.add("@" + (nativeConfigPath.resolve("native-image-args.txt")));
         nativeArgs.addAll(Lists.of("-cp", classPath));
 
+        if (currentPackage.project().kind() == ProjectKind.SINGLE_FILE_PROJECT) {
+            packageName = currentPackage.project().sourceRoot().toString().replace(".bal","");
```

Comment:
We can use `ProjectConstants.BLANG_SOURCE_EXT` instead of the `.bal` string

Reply:
Done